### PR TITLE
Fix mixed-content warning on SSL and add redirect.

### PR DESCRIPTION
**Pull request by bardiharborow**
Need I say more?

### CODE REVIEW

File: js/githubresume.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+if (window.location.protocol != 'https:') window.location.protocol = "https:";
+
```
Comment (u2) [inline js/githubresume.js]: I don't think we should force https, and also it will cause problem for the developer run on local.

File: index.html [hunk 2]
```diff
@@ -5,10 +5,10 @@
 	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
 	<meta name="keywords" content="cv, resume, hResume, github, community, free, fun, nice, useful" />
 	<meta name="description" content="GitHub resumes, generated by the community, for the community." />
-	<link rel="stylesheet" type="text/css" href="http://yui.yahooapis.com/2.7.0/build/reset-fonts-grids/reset-fonts-grids.css" media="all" /> 
+	<link rel="stylesheet" type="text/css" href="css/reset-fonts-grids.css" media="all" /> 
 	<link rel="stylesheet" type="text/css" href="css/resume.css" media="all" />
 	<link rel="stylesheet" type="text/css" href="css/print.css" media="print" />
-	<script type="text/javascript" src="http://code.jquery.com/jquery-1.8.1.min.js"></script>
+	<script type="text/javascript" src="https://code.jquery.com/jquery-1.8.1.min.js"></script>
 	<script type="text/javascript" src="js/mustache.js"></script>
```
Comment (u2) [inline index.html]: `//` may be better?

Comment (bardiharborow) [inline index.html]: "Now that SSL is encouraged for everyone
<https://www.eff.org/encrypt-the-web-report> and doesn’t have performance
concerns <https://istlsfastyet.com/>, *this technique is now an
anti-pattern*. If the asset you need is available on SSL, then *always* use
the https:// asset."
http://www.paulirish.com/2010/the-protocol-relative-url/

Comment (u2) [inline index.html]: Yes. I agree with you. I have leave a comment at #182, but forgot to remove this comment. Sorry.

File: js/githubresume.js (hunk 1 above)
Comment (u2) [inline js/githubresume.js]: And I think we should do it on the server side, what's your opinion?

File: index.html (hunk 2 above)
Comment (u2) [inline index.html]: Have merged #205. I will keep this PR open for the https redirect.

**Pull request closed without merging**